.changeset/shiny-crews-sleep.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kadena/react-ui": minor
+---
+
+Add variant option for different colors
```

Comment:
I don't think this is descriptive enough for someone reading release docs. Maybe something like "Added a new variant to apply different colors to the Divider component"

Reply:
Agreed, changed.